packages/theme/src/theme/_modals.scss

```diff
@@ -5,12 +5,42 @@
 	flex-direction: column;
 }
 
+/** IE-FIX */
+.modal-dialog {
```

Comment:
Can you merge with the previous class ? Those 2 blocks are for `.modal-dialog` and the second one overrides some properties of the first one.